.github/actions/publish-nuget/New-NugetPackage.ps1

```diff
@@ -0,0 +1,32 @@
+
+<#
+.SYNOPSIS
```

Comment:
I'd indent this [like here](url).

Reply:
Your url is broken. Also this is the result of the `comment-help` snippet in VS Code ([see](https://lombiq.atlassian.net/wiki/spaces/DEV/pages/43384918/PowerShell+code+conventions#:~:text=just%20start%20typing%20comment-help%20a%20line%20above%20the%20function%20and%20select%20it%20in%20the%20intellisense%20suggestions.)) from the official PowerShell extension so they probably know what they are doing.

Reply:
Funny, because [the example](https://docs.microsoft.com/en-us/powershell/scripting/developer/help/examples-of-comment-based-help?view=powershell-7.2#example-2-comment-based-help-for-a-script) I was referring to is also from the MS PS team. But ➕ 1️⃣ for not adjusting auto-generated snippets.